ansible_rulebook/cli.py

```diff
@@ -354,7 +354,7 @@ def main(args: List[str] = None) -> int:
     except KeyboardInterrupt:
         return 0
     except Exception as err:
-        logger.error("Terminating %s", str(err))
+        logger.error("Terminating: %s", str(err))
```

Comment:
@kaiokmo Can we catch well known Source Exceptions here

```
      except (SourcePluginNotFoundException, SourceFilterNotFoundException) as err:
            logger.error("Terminating: %s", str(err))
```

We need to add similar messages for other Source exceptions and catch them all here in one place. 
https://github.com/ansible/ansible-rulebook/blob/eeddfc05661205478bc276740bd7114a55ed9809/ansible_rulebook/exception.py#L94
https://github.com/ansible/ansible-rulebook/blob/eeddfc05661205478bc276740bd7114a55ed9809/ansible_rulebook/exception.py#L98
https://github.com/ansible/ansible-rulebook/blob/eeddfc05661205478bc276740bd7114a55ed9809/ansible_rulebook/exception.py#L102